rllib/core/rl_module/encoder.py

```diff
@@ -13,6 +13,9 @@
 
 # TODO (Kourosh): Find a better / more straight fwd approach for sub-components
 
+ENCODER_OUT = "embedding"
+STATE_IN = "state_in"
```

Comment:
`state` is super overloaded. Is it possible to call these `recurrent_state` or something?